src/pages/assets.tsx

```diff
@@ -0,0 +1,479 @@
+import ethLandscapeBlackGray from "@/public/assets/ethereum-logo-landscape-black-gray.png"
+import ethLandscapePurple from "@/public/assets/ethereum-logo-landscape-purple.png"
+import ethLandscapePurplePurple from "@/public/assets/ethereum-logo-landscape-purple-purple.png"
+import ethLandscapePurpleWhite from "@/public/assets/ethereum-logo-landscape-purple-white.png"
+import ethPortraitBlack from "@/public/assets/ethereum-logo-portrait-black.png"
+import ethPortraitBlackGray from "@/public/assets/ethereum-logo-portrait-black-gray.png"
+import ethPortraitPurple from "@/public/assets/ethereum-logo-portrait-purple.png"
+import ethPortraitPurplePurple from "@/public/assets/ethereum-logo-portrait-purple-purple.png"
+import ethWordmarkBlack from "@/public/assets/ethereum-wordmark-black.png"
+import ethWordmarkBlackGray from "@/public/assets/ethereum-wordmark-black-gray.png"
+import ethWordmarkPurple from "@/public/assets/ethereum-wordmark-purple.png"
+import ethWordmarkPurplePurple from "@/public/assets/ethereum-wordmark-purple-purple.png"
+import ethWordmarkPurpleWhite from "@/public/assets/ethereum-wordmark-purple-white.png"
+import finance from "@/public/finance_transparent.png"
+import future from "@/public/future_transparent.png"
+import hackathon from "@/public/hackathon_transparent.png"
+import hero from "@/public/home/hero.png"
+import heroPanda from "@/public/home/hero-panda.png"
+import impact from "@/public/impact_transparent.png"
+import infrastructure from "@/public/infrastructure_transparent.png"
+import merge from "@/public/upgrades/merge.png"
+import mergePanda from "@/public/home/merge-panda.png"
+import newRings from "@/public/upgrades/newrings.png"
+import oldShip from "@/public/upgrades/oldship.png"
+import wallet from "@/public/wallet.png"
+import whatIsEthereum from "@/public/what-is-ethereum.png"
+// import efLogo from "@/public/ef-logo.png"
+// import efLogoWhite from "@/public/ef-logo-white.png"
+// import ethGifCat from "@/public/eth-gif-cat.png"
+// import ethGifChalk from "@/public/eth-gif-chalk.png"
+// import ethGifSun from "@/public/eth-gif-sun.png"
+// import ethGifWaves from "@/public/eth-gif-waves.png"
+// import ethPortraitPurpleWhite from "@/public/assets/ethereum-logo-portrait-purple-white.png"
+// import leslieTheRhino from "@/public/upgrades/upgrade_rhino.png"
+
+import type { ChildOnlyProp } from "@/lib/types"
+
+const Row = (props: SimpleGridProps) => (
+  <SimpleGrid
+    templateColumns="repeat(auto-fit, minmax(min(288px, 100%), 1fr))"
+    mx={-4}
+    mb={8}
+    {...props}
+  />
+)
+
+const H2 = (props: ChildOnlyProp & HeadingProps) => (
+  <Heading
+    fontSize={{ base: "2xl", md: "3xl" }}
+    lineHeight={1.4}
+    mt={16}
+    mb={6}
+    {...props}
+  />
+)
+
+const H3 = (props: ChildOnlyProp) => (
+  <OldHeading
+    as="h3"
+    fontSize={{ base: "xl", md: "2xl" }}
+    lineHeight={1.4}
+    mb={0}
+    {...props}
+  />
+)
+
+const AssetsPage = () => {
+  const { t } = useTranslation("page-assets")
```

Comment:
@nhsz @pettinarip I'm missing something related to getting the i18n namespace working for this path... any idea where this needs to be updated to get the translations working?